### PR TITLE
Add lawyer-friendly hosted logo(s)

### PR DESCRIPTION
**Pull request by matthew-mcallister**
This adds a new "hosted-by-OCF" logo based on the old box badge, but this time in SVG format and using the font and penguin logo from the website.

The idea is to make something that looks decent, reproduce it in different color schemes, and use it as a drop-in replacement for the old logo. A potential minor concern is that I had to resize the badge by ~25% because the SVG doesn't look very good at 98x39.

I'll remove all the inkscape markup from the SVG before merging.

### CODE REVIEW

File: ocfweb/static/img/hosting-logos/ocf-hosted-penguin.svg [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+                 d="M 0,0 -9.921,-6.688 -7.197,2.862 0,5.724 Z"
+                 inkscape:connector-curvature="0" />
+            </g>
+          </g>
+        </g>
+      </g>
+      <text
+         xml:space="preserve"
+         style="font-style:normal;font-weight:normal;font-size:21.1427784px;line-height:0.75;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.52856946"
+         x="314.98428"
```
Comment (kpengboy) [inline ocfweb/static/img/hosting-logos/ocf-hosted-penguin.svg]: I think the penguin and the text are too close to each other at their narrowest points.

**Comment by jvperrin** [comment]
Nice, looks good, I'd put a little less bold on the "OCF" text, since it looks a bit too bold to me.

The failing test should be reasonably easy to change so that it checks that the returned images are the correct format (matching the file extension), so I think that would be best?

**Comment by tahabi** [comment]
Is there a sans serif option? I think that would look pretty good, matching the existing font scheme on the website.

File: ocfweb/static/img/hosting-logos/ocf-hosted-penguin.svg (hunk 1 above)
Comment (matthew-mcallister) [inline ocfweb/static/img/hosting-logos/ocf-hosted-penguin.svg]: Any better now?

**Comment by matthew-mcallister** [comment]
@jvperrin I changed the "OCF" from heavy to bold, which actually better matches the old logo upon comparison (kind of hurts the kerning at this resolution though). Sorry about the failing tests, I completely forgot to run them at first.

@tahabi What does the badge look like to you?

**Comment by tahabi** [comment]
screen shot 2017-03-07 at 3 48 43 am

**Comment by matthew-mcallister** [comment]
Hm, I guess this won't work out since not everybody has Roboto installed. I think the easiest and probably best solution is to just convert the text to a path before merging.

File: ocfweb/main/hosting_logos.py [hunk 2]
```diff
@@ -28,9 +28,13 @@
 
 @cache()
 def get_image(image):
-    if not re.match(r'^[a-z0-9_\-]+\.png$', image):
+    match = re.match(r'^[a-z0-9_\-]+\.(png|svg)$', image)
+    if not match:
         raise Http404()
 
+    # 'image/png' or 'image/svg'
+    content_type = 'image/png' if match.group(1) == 'png' else 'image/svg+xml'
```
Comment (chriskuehl) [inline ocfweb/main/hosting_logos.py]: instead of this can you do

```python
import mimetypes
mimetype, _ = mimetypes.guess_type(image)
```

**Comment by NickImpicciche** [comment]
Can we get this merged in soon? Very needed addition and seems like only minor changes are needed

**Comment by kpengboy** [comment]
Yeah, the text just needs to be converted into a path.

For further enhancement, we can make different color versions of this logo, and consider monkey-patching the current logos with these new logos so most everyone's banner changes.

**Comment by matthew-mcallister** [comment]
Yes, certainly, @kpengboy's suggestions are exactly my original plans. This
will take some minor design work so I'll update the PR as soon as it's
ready (won't take long).

On Apr 19, 2017 12:57 PM, "kpengboy" <notifications@github.com> wrote:

Yeah, the text just needs to be converted into a path.

For further enhancement, we can make different color versions of this logo,
and consider monkey-patching the current logos with these new logos so most
everyone's banner changes.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ocf/ocfweb/pull/268#issuecomment-295414012>, or mute
the thread

**Comment by matthew-mcallister** [comment]
OK, I subbed out the old "ocfbadge_mini8" logos with the light version and a new dark version of questionable design merit. I feel a little bad about replacing images with different sized ones, but, hey, a bigger logo is easier for visitors to see.

**Comment by matthew-mcallister** [comment]
Any thoughts on the dark penguin?

**Comment by matthew-mcallister** [comment]
This got the thumbs-up in-person from @jvperrin and @kpengboy.

**Pull request merged**